index.html

```diff
@@ -1336,58 +1336,73 @@ <h2>Dereferencing the Fragment</h2>
 
 	<section>
 		<h2>Examples</h2>
-		<p>Given the following <var>input <a>DID URL</a></var>:</p>
-		<pre class="example nohighlight">
-did:example:123456789abcdefghi#keys-1
-</pre>
-		<p>... and the following <var>resolved <a>DID document</a></var>:<p>
-		<pre class="example nohighlight">
-{
-	"@context": "https://www.w3.org/ns/did/v1",
-	"id": "did:example:123456789abcdefghi",
-	"verificationMethod": [{
+
+		<section>
+			<h3>Example: Dereferencing to verification method</h3>
+			<p>Given the following <var>input <a>DID URL</a></var>:</p>
+			<pre class="example nohighlight">
+	did:example:123456789abcdefghi#keys-1
+	</pre>
+			<p>... and the following <var>resolved <a>DID document</a></var>:<p>
+			<pre class="example nohighlight">
+	{
+		"@context": "https://www.w3.org/ns/did/v1.1",
+		"id": "did:example:123456789abcdefghi",
+		"verificationMethod": [{
+			"id": "did:example:123456789abcdefghi#keys-1",
+			"type": "Multikey",
+			"controller": "did:example:123456789abcdefghi",
+			"publicKeyMultibase": "z6MkmM42vxfqZQsv4ehtTjFFxQ4sQKS2w6WR7emozFAn5cxu"
+		}],
+		"service": [{
+			"id": "did:example:123456789abcdefghi#agent",
+			"type": "AgentService",
```

Comment:
Wondering if we should be using existing defined types found in the extensions https://www.w3.org/TR/did-extensions-properties/#service-types